File: MovieWebsite-fontend/src/app/dtos/user/order.dto.ts
import{
    IsNumber
 }from 'class-validator'
 
 export class OrderDTO{
     
    @IsNumber()
     user_id: number;

    @IsNumber()
     price: number;

    
     constructor(data: any){
        this.user_id= data.user_id 
        this.price= data.price        
     }
 }

File: MovieWebsiteJA-tin/MovieWebsiteJA-tin/MovieWebsite-backend/src/main/java/com/project/MovieWebsite/dtos/OrderDTO.java
package com.project.MovieWebsite.dtos;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.*;

@Data
@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor

public class OrderDTO {

    @JsonProperty("user_id")
    private int userId;

    @JsonProperty("price")
    private double price;

}
